src/sdk/namespace_fs.js

```diff
@@ -1321,6 +1326,11 @@ class NamespaceFS {
                 await this.append_to_migrate_wal(file_path);
             }
         }
+        if (params.tagging) {
+            for (const { key, value } of params.tagging) {
+                fs_xattr[XATTR_TAG + key] = value;
+            }
+        }
```

Comment:
I saw that in previous assigns they used `fs_xattr = Object.assign(fs_xattr || {}, ...` since there might be a case where `fs_xattr` is `undefined`.